README.md

```diff
@@ -343,9 +343,32 @@ foreach ($annotation->faces() as $key => $face) {
 }
 ```
 
+## Caching Access Tokens
+
+By default the library will use a simple in-memory caching implementation, however it is possible to override this behavior by passing a [PSR-6](http://www.php-fig.org/psr/psr-6/) caching implementation in to the desired client.
+
+The following example takes advantage of [Symfony's Cache Component](https://github.com/symfony/cache).
+
+```php
+<?php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Storage\StorageClient;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+
+$cacheDir = '/tmp/google-cloud-cache';
+// Make sure to set the proper permissions to restrict access to tokens.
+@mkdir($cacheDir, 0600);
```

Comment:
I know what you mean, but this is not about a `cache` in a normal sense. It caches the auth tokens.

I feel it's irresponsible if we just say "use whatever cache implementation as you need", and not mentioning a security risk of using filesystem cache for the token cache.

Can you add security note instead?